src/sentry/static/sentry/app/views/organizationEventsV2/modalPagination.jsx

```diff
@@ -75,13 +76,22 @@ ModalPagination.propTypes = {
 const StyledLink = styled(Link, {shouldForwardProp: isPropValid})`
   color: ${p => (p.disabled ? p.theme.disabled : p.theme.gray3)};
   font-size: ${p => p.fontSizeMedium};
+  text-align: center;
   padding: ${space(0.5)} ${space(1.5)};
   ${p => (p.isLast ? '' : `border-right: 1px solid ${p.theme.borderDark};`)}
   ${p => (p.disabled ? 'pointer-events: none;' : '')}
+
+  @media(max-width: ${theme.breakpoints[0]}) {
```

Comment:
Do we want to import theme rather than use the convention on the lines above, e.g. `${p => p.theme.breakpoints}`?

Reply:
I've seen both patterns used in the code base. I can switch the the callback pattern as we use that more broadly.